app/client/components/forms/BasicFormField.tsx

```diff
@@ -1,17 +1,43 @@
-import { Field, Label } from "@headlessui/react";
+import { Field } from "@headlessui/react";
 import { FieldHandler } from "@reactivated";
 import React from "react";
+import { BasicDescription } from "./BasicDescription";
+import { BasicLabel } from "./BasicLabel";
 import { BasicWidget } from "./BasicWidget";
 
 interface BasicFormFieldProps {
   field: FieldHandler;
+  disabled?: boolean;
+  checked?: boolean;
+  onCheckboxChange?: (field: FieldHandler, value: boolean) => void;
 }
 
-export const BasicFormField = ({ field }: BasicFormFieldProps) => {
+export const BasicFormField = ({
+  field,
+  disabled = undefined,
+  checked = undefined,
+  onCheckboxChange = undefined,
+}: BasicFormFieldProps) => {
   return (
-    <Field className="mb-3">
-      <Label className="label font-semibold">{field.label}</Label>
-      <BasicWidget field={field} />
+    <Field
+      className="mb-4 flex flex-col"
+      disabled={disabled === undefined ? field.disabled : disabled}
+    >
+      <div
+        className={`flex ${
+          field.tag === "django.forms.widgets.CheckboxInput"
```

Comment:
It might be a good idea to use `clsx` here; it will be more readable

Reply:
Done